lib/api/article/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface fetchArticleByIdParams {
+  articleId: string;
+}
+
+export interface fetchArticleCommentsByIdParams {
+  articleId: string;
+}
```

Comment:
```suggestion
export type ArticleId = {
  articleId: string;
};

export type fetchArticleByIdParams = ArticleId;
export type fetchArticleCommentsByIdParams = ArticleId;
```

타입도 변수처럼 재사용이 가능해서 이런 방법도 있습니다. 

아니면
```typescript
export type ArticleId = {
  articleId: string;
};
```
타입 하나만 만들어서 써도 되긴 합니다 :-)